cli/azd/cmd/auth_logout.go

```diff
@@ -10,15 +10,19 @@ import (
 	"github.com/azure/azure-dev/cli/azd/cmd/actions"
 	"github.com/azure/azure-dev/cli/azd/pkg/account"
 	"github.com/azure/azure-dev/cli/azd/pkg/auth"
+	"github.com/azure/azure-dev/cli/azd/pkg/input"
 	"github.com/azure/azure-dev/cli/azd/pkg/output"
 	"github.com/spf13/cobra"
 )
 
-func newLogoutCmd() *cobra.Command {
+func newLogoutCmd(parent string) *cobra.Command {
 	return &cobra.Command{
 		Use:   "logout",
 		Short: "Log out of Azure.",
 		Long:  "Log out of Azure",
+		Annotations: map[string]string{
+			loginCmdParentAnnotation: parent,
+		},
 	}
 }
```

Comment:
What is the parent used for? I see you are setting this custom annotation but not using it anywhere?

Reply:
That's a keen observation! The initial plan was to hide and issue a warning, that's what the annotation would be used for. I'm a little hesitant on that given the use of `azd login` in the Azure SDK. I'm figuring out this part with PM and team

Reply:
Added as follow-up in: https://github.com/Azure/azure-dev-pr/issues/1497#issuecomment-1487290995